### PR TITLE
Feature/alpayk nuc117 transfer confirm daialog

### PR DESCRIPTION
**Pull request by alpaykuru**
Screenshot from 2019-05-03 11-36-05

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -39,6 +39,13 @@ module.exports = {
               "components/slider.md"
             ]
           },
+          {
+            title: "Dialogs",
+            collapsable: true,
+            children: [
+              "components/dialogs/confirmDialog.md",
```
Comment (ercang) [inline docs/.vuepress/config.js]: according to the other filenames, this should be confirm-dialog.md

File: docs/components/dialogs/confirmDialog.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+sidebarDepth: 0
+---
+
+# ConfirmDialog
+
+This is an overview of the most common usage of ConfirmDialog.
+
+## Live Demo
+
+<DocExampleBox codeBox="https://codesandbox.io/s/m7xmj8xnxj?module=%2Fsrc%2FApp.vue">
```
Comment (ercang) [inline docs/components/dialogs/confirmDialog.md]: you can remove the codex url and also add live demo prop.
< DocExampleBox :liveDemoMode="true" >

File: src/components/dialogs/ConfirmDialog.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  <ModalDialog v-if="isModalVisible" class="vxp-confirm-dialog" @close="close(false)">
+    <span class="vxp-confirm-dialog__header" slot="header">{{ title }}</span>
+
+    <div v-if="message" class="vxp-confirm-dialog__body" slot="body">
+      <div>{{ message }}</div>
+    </div>
+
+    <div class="vxp-confirm-dialog__footer" slot="footer">
+      <Button class="nvw-confirm-dialog__footer__cancel-button" :text="cancelButtonText" @tap="close(false)" />
+      <Button class="nvw-confirm-dialog__footer__ok-button" :text="okButtonText" @tap="close(true)" />
```
Comment (ercang) [inline src/components/dialogs/ConfirmDialog.vue]: css classnames should start with vxp

File: src/components/dialogs/ConfirmDialog.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<style lang="scss">
+@import '../../assets/styles/components/dialogs';
+.vxp-confirm-dialog {
+  &__footer {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    &__cancel-button,
+    &__ok-button {
+      @include nvw-dialog-button;
```
Comment (ercang) [inline src/components/dialogs/ConfirmDialog.vue]: class name should be different

File: src/plugins/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import ConfirmDialog from '../components/dialogs/ConfirmDialog';
+
+const NvwPlugin = {
```
Comment (Serhatkemal) [inline src/plugins/index.js]: Add unit tests for ConfirmDialogPlugin.

**Pull request merged**